game/src/main/scala/io/aigar/game/Game.scala

```diff
@@ -21,13 +20,15 @@ class Game(val id: Int, playerIDs: List[Int], val duration: Int = Game.DefaultDu
   val startTime = GameThread.time
   var tick = 0
 
-  def update(deltaSeconds: Float): MutableList[ScoreModification] = {
-    players.foreach { player => player.update(deltaSeconds, grid, players) }
-    viruses.update(grid, players)
-    val scoreModifications = resources.update(grid, players)
+  def update(deltaSeconds: Float): List[ScoreModification] = {
+    var modifications = List[ScoreModification]()
+
+    players.foreach { player => modifications :::= player.update(deltaSeconds, grid, players) }
```

Comment:
I've put flatten since it was as `List `of `List`! Thanks it is way much cleaner!